app/test_split_integer.py

```diff
@@ -1,21 +1,34 @@
-from app.split_integer import split_integer
-
+import unittest
```

Comment:
You need to use PyTest according to Readme *hint: you don't need to import anything

Reply:
Oh, didn't pay attention